docs/quick_start.md

```diff
@@ -180,19 +158,19 @@ Note: *If any changes are made to the dependencies section of the `package.json`
     "test": "react-scripts test",
     "prebuild": "electron-rebuild",
     "build": "react-scripts build",
-    ...    
+    ...
 }
```

Comment:
Maybe check these are the scripts? Update them if they have been changed even though this is just an example?

Reply:
They seem to be up to date